styles.css

```diff
@@ -0,0 +1,180 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,500;0,700;1,300&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Open Sans";
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+
+header {
+  display: flex;
+  color: #e5e7eb;
+  font-size: 18px;
+  align-items: center;
+  gap: 24px;
+  padding: 16px 15%;
+  background-color: #1f2937;
+  flex: 1;
+}
+
+.logo {
+  margin-right: auto;
+  font-size: 24px;
+  color: #f9faf8;
+  font-weight: bold;
+}
+
+main {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+}
+
+.intro {
+  display: flex;
+  background-color: #1f2937;
+  padding: 100px 15%;
+  flex: 1;
+  gap: 18px;
+}
+
+.intro-text,
+.intro-image {
+  flex: 1;
+}
+
+.intro-text h1 {
+  font-size: 48px;
+  color: #f9faf8;
+  font-weight: 900;
+}
+
+.intro-text p {
+  font-size: 18px;
+  color: #e5e7eb;
+}
+
+.intro-button {
+  background-color: #3882f6;
+  color: white;
+  padding: 8px 32px;
+  border-radius: 8px;
+  font-size: 16px;
+  font-weight: bold;
+  margin-top: 12px;
+  border: none;
+}
+
+.intro-image {
+  background-color: #6d747d;
+  color: #e5e7eb;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.info {
+  padding: 32px 15%;
+  margin-bottom: 64px;
+  flex: 1;
+}
+
+.info h2 {
+  font-size: 36px;
+  font-weight: bold;
+  color: #1f2937;
+  text-align: center;
+  margin-bottom: 32px;
+}
+
+.info ul {
+  display: flex;
+  justify-content: center;
+  gap: 48px;
+}
+
+.info li {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  max-width: 160px;
+}
+
+.info p,
+.info div {
+  flex: 1 1 auto;
+  text-align: center;
+  color: #6d747d;
+}
+
+.info-box {
+  border: #3882f6 solid 4px;
+  height: 160px;
+  width: 160px;
+  border-radius: 16px;
+}
+
+.quote {
+  background-color: #e5e7eb;
+  flex: 1;
+  padding: 100px 23%;
+}
+
+.quote p {
+  font-size: 32px;
+  color: #1f2937;
+  font-style: italic;
+  font-weight: lighter;
+}
+
+.quote .second-paragraph  {
+  text-align: right;
+  font-size: 24px;
+  font-weight: bold;
+  font-style: normal;
+}
+
+.signup-box {
+  margin: 100px 180px;
+  background-color: #3882f6;
+  padding: 48px 156px;
+  border-radius: 10px;
+  color: #ffffff;
+  display: flex;
+  justify-content: space-between;
+}
+
+.signup-button {
```

Comment:
please don't repeat yourself (DRY),  `intro-button` class and this class can merge in to one single class as this is sharing the same property